packages/@tinacms/cli/src/buildTina/attachDatabase.ts

```diff
@@ -45,7 +45,8 @@ export const attachDatabase = async (
     filename: 'database',
     allowedTypes: ['js', 'jsx', 'tsx', 'ts'],
   })
-  if (inputFile && !_options.useLocalDatabase) {
+  // if (inputFile && !_options.useLocalDatabase) {
```

Comment:
Not sure what other reasons there are for this check, just disabled i for now

Reply:
The useLocalDatabase option is to ensure that when running `yarn dev`, the user defined database is not used. When you run build or start and database.ts is present, then that database is used.